<div id="error_explanation">
  <h2>Следующие ошибки помешали сохранению:</h2>

  <ul>
    <% resource.errors.full_messages.each do |message| %>
        <li><%= message %></li>
    <% end %>
  </ul>
</div>
